### PR TITLE
Added to Workflow Troubleshooting

### PR DESCRIPTION
**Pull request by danielholdsworth**
# Description
Added a reminder that jobs that are 90 days or older cannot be re-run.

# Reasons
A customer reached out with an issue where they could not re-run a job, the job was over 90 days old so a re-run is not possible but this is not stated in our docs, only in a discuss article from 2020.

### CODE REVIEW

File: jekyll/_cci2/workflows.md [hunk 1]
```diff
@@ -641,6 +641,8 @@ to discern what might be failing.
 
 It has been observed that in some cases, a failure happens before the workflow runs (during pipeline processing). In this case, re-running the workflow will fail even though it was succeeding before the outage. To work around this, push a change to the project's repository. This will re-run pipeline processing first, and then run the workflow.
 
+Also note that jobs and workflows that are 90 days or older are unable to be re-run.
```
Comment (kelvintaywl) [inline jekyll/_cci2/workflows.md:644]: Nice! 🙌 

Just a suggestion/opinion, as I think active voice may be easier to read in this case!
> https://developers.google.com/tech-writing/one/active-voice

```suggestion
Also, please note that you cannot re-run jobs and workflows that are 90 days or older.
```

Comment (danielholdsworth) [reply jekyll/_cci2/workflows.md]: Agreed!

**Review by kbatuigas — APPROVED** [review]
Thanks, @danielholdsworth !

**Comment by kbatuigas** [comment]
Hey @danielholdsworth ! Sorry for the delay in getting back to you on this, but we've actually been seeing issues where PRs to the docs repo but the builds never start, which is the case here. Could I just double check what your workflow was for this PR? Did you happen to make the changes via git/the console or was it just directly here on GitHub after you forked the repo? We haven't quite figured out what exactly is causing some pipelines to build and others not, but it may help to know the flow you followed. Thanks!

**Comment by danielholdsworth** [comment]
> 

Hey @kbatuigas ! I actually made the edit here in the GH UI as it was such a small change

**Comment by rosieyohannan** [comment]
Hey @danielholdsworth could you try the following: Log out of GitHub and CircleCI, log back in, then push an empty commit on this branch using `git commit --allow-empty`? Thank you and sorry for all the bother!

**Comment by danielholdsworth** [comment]
> 

All done!

**Comment by rosieyohannan** [comment]
Oh no well that didn't work!

**Comment by rosieyohannan** [comment]
I think we'll have to make a separate PR to make this change. No idea why it won't build.

**Comment by danielholdsworth** [comment]
> I think we'll have to make a separate PR to make this change. No idea why it won't build.

ok no worries, I'll close the PR and re-do it!

**Pull request closed without merging**